app/assets/javascripts/admin/edition_publishing.js

```diff
@@ -217,3 +217,21 @@ jQuery(function($) {
 
   SetTopicsFromPolicy.init();
 }(jQuery));
+
+(function($) {
+  var $policyInput = $('#edition_policy_content_ids'),
+      $importantCheckbox = $('#edition_important');
+
+  var toggleCheckboxIfPoliciesSelected = function() {
+    if ($importantCheckbox >= 1) {
+      $importantCheckbox.prop('disabled', false)
+    } else {
+      $importantCheckbox.prop('disabled', true)
+    }
+  }
+
+  if ($policyInput && $importantCheckbox) {
```

Comment:
I believe this will always be true because the objects will exist. You are safer to do something like `if($policyInput.length > 0 && $importantCheckbox.length > 0)`. Which makes sure that there are actually matched elements